### DISCUSSION TITLE
Custom UI components for the designer

### DISCUSSION THREAD

**Discussion by abdallahSQaradeh** [elsa-workflows/elsa-core#4855] [Q&A]
Is there a way to write our custom UI for the designer? What are the supported frameworks?

**Comment by MeloHenrique** [comment]
Hi @abdallahSQaradeh,

I realize this is an older question, but from my experience, it is possible to add custom UI components using Blazor components. The main challenge, however, is the current lack documentation.

**Reply by dupies** [reply]
I have the same [question ](https://github.com/elsa-workflows/elsa-core/discussions/5931)
